docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,234 @@
+
+PaperQA builds the index every time a question is asked and new files are present. So running this will build the index for you.
+
+You don’t need any api keys for building the index, but you do need them to answer questions.
+
+This process is quick for small portions of the whole document’s dataset, but can take ~3hs for the whole of it.
+
+If you want to use the index we built with the whole dataset, you can get it here. Make sure you put it in the correct folder and change the index name to `"lfrqa_science_index_complete"`
+
+```python
+from paperqa import Settings, ask
+
+settings = Settings()
+settings.agent.index.name = "lfrqa_science_index"
+settings.agent.index.paper_directory = (
+    "data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa"
+)
+settings.agent.index.index_directory = (
+    "data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa_index"
+)
+
+settings.agent.index.manifest_file = "manifest.csv"
+
+settings.parsing.use_doc_details = False
+settings.parsing.defer_embedding = True
+
+settings.agent.index.concurrency = 10000
+settings.agent.index.batch_size = 10000
+
+answer_response = (
+    ask(
+        "$5^n+n$ is never prime?",
+        settings=settings,
+    ),
+)
+```
+
+After this runs, you will get an answer!
+
+## Benchmark!
+
+After you have built the index, you are ready to run the benchmark.
+
+Copy the following into a file `gradable.py` and run it. You can also use the parameter num_questions in `LFRQATaskDataset` so you can make quick tests.
+
+```python
+import asyncio
+from ldp.agent import SimpleAgent
+from ldp.alg.callbacks import MeanMetricsCallback
+from ldp.alg.runners import Evaluator, EvaluatorConfig
+from paperqa import Settings
+from paperqa.agents.task import LFRQATaskDataset
+
+
+async def evaluate() -> None:
+    settings = Settings()
+    settings.agent.index.name = "lfrqa_science_index"
+    settings.agent.index.paper_directory = (
+        "data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa"
+    )
+    settings.agent.index.index_directory = (
+        "data/rag-qa-benchmarking/lfrqa/science_docs_for_paperqa_index"
+    )
+
+    dataset = LFRQATaskDataset(
+        data_path="data/rag-qa-benchmarking/lfrqa/questions.csv",
+        num_questions=4,
+        settings=settings,
+    )
+    metrics_callback = MeanMetricsCallback(eval_dataset=dataset)
+
+    evaluator = Evaluator(
+        config=EvaluatorConfig(batch_size=3),
+        agent=SimpleAgent(),
+        dataset=dataset,
+        callbacks=[metrics_callback],
+    )
+    await evaluator.evaluate()
+
+    print(metrics_callback.eval_means)
+
+
+if __name__ == "__main__":
+    asyncio.run(evaluate())
+```
+
+After running this, you will get a result like this:
+
+```
+{'reward': 0.5, 'truncation_rate': 0.0, 'avg_value': 0.0, 'num_steps': 4.0, 'failures': 0.0, 'total_paper_count': 8.5, 'relevant_paper_count': 5.0, 'evidence_count': 6.0, 'paperqa_won': 0.89}
```

Comment:
`paperqa_won` -- this isn't part of `MeanMetricsCallback`, can you remove this one?

Reply:
But I added things to the callback. I think this one particularly is quite useful to quickly see how it performed.